site/content/docs/developer-guide/release/plugin-updates.md

```diff
@@ -15,7 +15,7 @@ This manual operation looks like:
 
 > **Note:** Ideally, the specified `version:` field should match the release tag
 of the plugin. This helps users and maintainers to easily identify which
-version of the plugin they have installed.
+version of the plugin they have installed. Also, the prefix 'v' is required.
```

Comment:
I don't think it belongs here. We don't talk about format of the file in this doc really.